### ISSUE TITLE
"The handlebars dependency defined in package-lock.json has a known moderate severity security vulnerability in version range < 4.0.0 and should be updated."

### ISSUE THREAD

**Issue by YajJackson** [angular/angular-cli#8544]
### Versions
```
Output from: `ng --version`:
Angular CLI: 1.5.2
Node: 6.11.3
OS: win32 x64
Angular: 5.0.2
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

@angular/cli: 1.5.2
@angular-devkit/build-optimizer: 0.0.33
@angular-devkit/core: 0.0.20
@angular-devkit/schematics: 0.0.36
@ngtools/json-schema: 1.1.0
@ngtools/webpack: 1.8.2
@schematics/angular: 0.1.5
typescript: 2.4.2
webpack: 3.8.1
```

### Repro steps
```
Step 1: Run `ng new <ExampleName>`
Step 2: View handlebars.js version 1.3.0 dependency in package-lock.json
```

### Observed behavior
```
Github flags this as a vulnerable dependency.
```

### Desired behavior
```
Update handlebars.js version dependency from 1.3.0 to 4.0.11
Reduce vulnerability out of the box.
```

**Comment by clydin** [comment]
Dupe of https://github.com/angular/angular-cli/issues/8534

**Comment by YajJackson** [comment]
@clydin I had updated @angular/cli under an hour prior to posting this, but notice a few people saying the issue was resolved.

**Comment by brunolm** [comment]
I updated @angular/cli to 1.5.2, removed node_modules, installed all again, still installed old handlebars version.

Is there an workaround for now?

**Comment by benc-uk** [comment]
CLI 1.52 doesn't fix it. There has been [a fix](https://github.com/angular/angular-cli/pull/8535)  - not sure what release it will surface in, 1.5.3 perhaps

**Comment by YajJackson** [comment]
@brunolm [this](https://github.com/YajJackson/NG5/commit/b89fdb0663e1bf429ba7934334cdb96904784b2e) has been my workaround.

**Comment by tigercosmos** [comment]
@YajJackson  It will only change `.lock` this time, and will become back in next time when `npm install`.
Not suggest to do that.

**Comment by brunolm** [comment]
@angular/cli 1.5.3 fixes it. https://github.com/angular/angular-cli/commits/v1.5.3

- Delete `node_modules`
- `npm i -D @angular/cli@1.5.3`
- `npm i`

https://github.com/brunolm/angular-how-to/pull/8

**Comment by filipesilva** [comment]
Duplicate of #8521, fixed by #8535.

**Comment by commercialsuicide** [comment]
I have tried Brunolm's solution, but when I run Angular, it fails to compile, the only logs I see:

> ERROR in ./src/client/main.ts
> Module build failed: [object Object]
>  @ multi webpack-dev-server/client?http://0.0.0.0:0 ./src/client/main.ts
> ERROR in ./src/client/polyfills.ts
> Module build failed: [object Object]
>  @ multi ./src/client/polyfills.ts

Tried with angular CLI 1.5.3 (locally and globally)
And another try with  angular CLI 1.6.0 (locally and globally)

**Comment by bofcarbon1** [comment]
I got these errors deploying React applications to github. Most of the issues were in the package-lock.json file. I use npm to build and run my React apps. There is a handy tool called 'npm-check'. Installing and running 'npm-check' will list outdated dependency libraries along with the npm commands to get the latest version. Its pretty cool. I updated my outdated scripts and that resolved the issue.

This post is a bit old but for those that come upon it and look for threads here you are.
Stack overflow won't let me share this until I get a 50 rep and that hasn't happened in
3 years since the new assholes took over and made it all about popularity.
